### PR TITLE
Buttons component added

### PR DESCRIPTION
**Pull request by JatuFaique**
Please review my buttons component,
Added Buttons style
Screenshot 2022-02-03 at 12-42-53 Buttons
s

### CODE REVIEW

File: button/buttons.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  align-items: flex-start;
+  gap: 8px;
+}
+
+.btn-section {
+  padding: 8px;
+}
+
+button {
+  padding: 8px 14px;
```
Comment (zabihshaik) [inline button/buttons.css]: Better to use rems instead of px

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just look at a comment

File: buttons.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  color: white;
+}
+
+.btn-secd {
+  border: none;
+  background-color: var(--bright-navy-blue);
+  color: white;
+}
+
+.btn-succ {
```
Comment (naazneen) [inline buttons.css]: Names could have been more specific.

**Review by naazneen — COMMENTED** [review]
Seems good overall

**Pull request merged**